api/valor_api/schemas/geometry.py

```diff
@@ -12,961 +13,542 @@
+    value : list[list[tuple[int | float, int | float]]]
+        A list of coordinates describing the MultiLineString.
 
-        Returns
-        ----------
-        float | int
-            The width of the geometry.
-        """
-        return self.boundary.width
-
-    @property
-    def height(self):
-        """
-        Returns the height of the geometry.
+    Raises
+    ------
+    ValueError
+        If the value doesn't conform to the type.
+    """
 
-        Returns
-        ----------
-        float | int
-            The height of the geometry.
-        """
-        return self.boundary.height
+    value: list[list[tuple[int | float, int | float]]]
+    model_config = ConfigDict(extra="forbid")
 
-    def wkt(self, partial: bool = False) -> str:
-        """
-        Returns the well-known text (WKT) representation of the object.
+    @model_validator(mode="before")
+    @classmethod
+    def deserialize_valor_type(cls, values: Any) -> Any:
+        return deserialize(class_name=cls.__name__, values=values)
 
-        Parameters
-        ----------
-        partial : bool
-            Whether to return the full WKT string or not.
+    @field_validator("value")
+    @classmethod
+    def validate_value(cls, v):
+        validate_type_multilinestring(v)
+        return v
 
-        Returns
-        ----------
-        str
-            The WKT representation of the shape.
-        """
-        polys = [self.boundary.wkt(partial=True)]
-        if self.holes:
-            for hole in self.holes:
-                polys.append(hole.wkt(partial=True))
-        wkt_format = f"({', '.join(polys)})"
-        if partial:
-            return wkt_format
-        return f"POLYGON {wkt_format}"
-
-    def offset(self, x: float = 0, y: float = 0):
-        """
-        Translates the geometry by an offset.
+    @classmethod
+    def load(cls, geojson: dict):
+        """Load MultiLineString from GeoJSON dictionary."""
+        geometry = GeoJSON(**geojson).geometry
+        if type(geometry) is not MultiLineString:
+            raise TypeError(f"GeoJSON is for a different type '{geojson}'.")
+        return geometry
+
+    def dump(self) -> dict[str, str | list[list[list[int | float]]]]:
+        """Dump MultiLineString to GeoJSON dictionary."""
+        return {
+            "type": "MultiLineString",
+            "coordinates": [
+                [list(point) for point in line] for line in self.value
+            ],
+        }
 
-        Parameters
-        ----------
-        x : int, default=0
-            The x-axis offset.
-        y : int, default=0
-            The y-axis offset.
-        """
-        self.boundary.offset(x, y)
-        if self.holes:
-            for idx in range(len(self.holes)):
-                self.holes[idx].offset(x, y)
+    @classmethod
+    def loads(cls, geojson: str):
+        """Load MultiLineString from GeoJSON string."""
+        return cls.load(json.loads(geojson))
+
+    def dumps(self) -> str:
```

Comment:
dumps -> to_json?

Reply:
I like this. 
load -> from_dict
loads -> from_json
dump -> to_dict
dumps -> to_json

Reply:
Original intent was to follow the convention of python's `json` library